File: 2017-10-04 Més Python/p4.py
# Suma d'una seqüència de nombres
# (recordeu que cal picar Control+D per finalitzar l'entrada al terminal)

# Compte: En Python no hi ha while (cin >> x).
# Quan read() no pot llegir, retorna un valor especial anomenat None.

from jutge import read

s = 0
x = read(int)
while x is not None:
    s = s + x
    x = read(int)
print(s)

File: 2017-09-26 Seqüènciem!/p4.cc
// Nombre d'elements successius repetits en una seqüència d'enters.

#include <iostream>
using namespace std;

int main() {
    int r = 0;
    int a;
    if (cin >> a) {
        int b;
        while (cin >> b) {
            if (b == a) r = r + 1;
            a = b;
        }
    }
    cout << r << endl;
}
